apps/team-awesome-web/src/components/button/Button.styled.ts

```diff
@@ -26,6 +26,14 @@ export const Button = styled.button<ButtonProps>`
       color: ${({ theme }) => theme.colorSecondary};
     `}
 
+  :hover {
+    background-color: ${({ theme }) => theme.backgroundSecondary};
+  }
+
+  :active {
+    transform: translate(4px, 2px);
```

Comment:
Is this a 'pressing animation'? Nice!

Reply:
Yes!!! You can do really cool stuff with this